staging/vhost-device-console/README.md

```diff
@@ -0,0 +1,133 @@
+# vhost-device-console - Console emulation backend daemon
+
+## Description
+
+This program is a vhost-user backend that emulates a VirtIO Console device.
+The device's binary takes as parameter a socket, a socket number which is the
+number of connections, commonly used across all vhost-devices to communicate
+with the vhost-user frontend devices, and the backend type "nested" or
+"network".
+
+By using the "nested" backend, vhost-device-console, as soon as, the guest
```

Comment:
This is a minor comment. How about rewrite it as:
```suggestion
The nested backend allows input/output to the guest console through the current terminal.
```

Reply:
Fully agree, nice touch! I just needed to adjuct it a bit for the other backend.